Sources/JWTKeychain/Support/UserAuthenticating.swift

```diff
@@ -65,39 +71,38 @@ public class UserAuthenticator: UserAuthenticating {
         }
 
         let name = data[User.Keys.name]?.string
-        let email = data[User.Keys.email]?.string
 
-        return try user.update(
-            email: email.map(Valid.init),
+        try user.update(
+            email: nil,
```

Comment:
Sorry if I'm misunderstanding, but what are the reasons for this change? How do we handle email updates?